.github/workflows/storage_visualization.yaml

```diff
@@ -6,6 +6,8 @@ on:
     - cron: "0 0 * * 0"
   # Run on demand.
   workflow_dispatch:
+  push:
+    branches: ['add-storage-viz-reqs']
```

Comment:
Debugging remnant?

```suggestion
```

Reply:
Just removed, didn't mean to tag you before I had removed that.